lib/Transformer.js

```diff
@@ -458,17 +475,29 @@ export class Transformer {
   }
 
   _getIdForTerm({term, plural}) {
-    const id = this.termToId.get(term);
+    let id;
+    // check keywords table
+    if(this.termToId.has(term)) {
+      id = this.termToId.get(term);
+    }
```

Comment:
The previous code avoided the extra lookup in the map, now the map is checked twice. Instead, just use the old code and check for `undefined` (which is what is already happening below). Also, the keywords table isn't used here, the `termToId` table is, so the comment is a bit off.